test/test_target.py

```diff
@@ -408,6 +408,52 @@ def test_pyopencl_execution_numpy_handling(ctx_factory):
     assert x[0] == 5.
 
 
+@pytest.mark.parametrize("target", [lp.PyOpenCLTarget, lp.ExecutableCTarget])
+def test_complex_support(ctx_factory, target):
```

Comment:
Maybe move to `test_expression.py`? Could go either way on this.

Reply:
Sure, done.  66990e1